app/views/pages/accessibility.html.erb

```diff
@@ -2,142 +2,180 @@
+          "listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)",
+        ]
+      } %>
+
+      <p class="govuk-body">
+        We’ve also made the website text as simple as possible to understand.
+      </p>
+
+      <p class="govuk-body">
+        <%= link_to 'AbilityNet', 'https://mcmw.abilitynet.org.uk/', target: :blank, class: 'govuk-link' %> has advice on making your device easier to use if you have a disability.
+      </p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "How accessible this website is",
+        margin_bottom: 4,
+        heading_level: 2,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        We know some parts of this website are not fully accessible for the following reasons:
+      </p>
+
+      <%= render "govuk_publishing_components/components/list", {
+        visible_counters: true,
+        items: [
+          "Find open data links to several external websites belonging to other government organisations. We cannot guarantee that these websites meet our accessibility requirements.",
+          "Some data sets have broken links.",
+        ]
+      } %>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Feedback and contact information",
+        margin_bottom: 4,
+        heading_level: 2,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        If you need any part of this service in a different format like large print, audio recording or braille, please contact the data set publisher.
+        Their contact details should be on the data set’s page.
+      </p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Reporting accessibility problems with this website",
+        margin_bottom: 4,
+        heading_level: 2,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        We’re always looking to improve the accessibility of this website.
+        If you find any problems not listed on this page or think we’re not meeting accessibility requirements, please select ‘Report a problem’ on the data.gov.uk <%= link_to 'Support page', '/support', target: :blank, class: 'govuk-link' %>.
+      </p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Enforcement procedure",
+        margin_bottom: 4,
+        heading_level: 2,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).
+        If you’re not happy with how we respond to your complaint, <%= link_to 'contact the Equality Advisory and Support Service (EASS)', 'https://www.equalityadvisoryservice.com/', target: :blank, class: 'govuk-link' %>.
```

Comment:
[We shouldn't open links in new tabs / windows by default](https://gds-way.cloudapps.digital/manuals/programming-languages/html.html#buttons-vs-links):

```suggestion
        If you’re not happy with how we respond to your complaint, <%= link_to 'contact the Equality Advisory and Support Service (EASS)', 'https://www.equalityadvisoryservice.com/', class: 'govuk-link' %>.
```